influxdb_iox/src/commands/debug/parquet_to_lp.rs

```diff
@@ -0,0 +1,73 @@
+//! This module implements the `parquet_to_lp` CLI command
+use std::{io::BufWriter, path::PathBuf};
+
+use observability_deps::tracing::info;
+use snafu::{ResultExt, Snafu};
+
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display("Can not {} output  file '{:?}': {}", operation, path, source))]
+    File {
+        operation: String,
+        path: PathBuf,
+        source: std::io::Error,
+    },
+    #[snafu(display("Error converting: {}", source))]
+    Conversion {
+        source: parquet_to_line_protocol::Error,
+    },
+    #[snafu(display("Can not flush output: {}", message))]
+    Flush {
+        // flush error has the W writer in it, all we care about is the error
+        message: String,
+    },
+}
+
+/// Convert IOx parquet files into InfluxDB Line Protocol format
```

Comment:
Parquet should probably be capitalized here too, and in this spot you've capitalized Line Protocol but I don't see that capitalized in official docs? https://docs.influxdata.com/influxdb/v2.4/reference/syntax/line-protocol/

```suggestion
/// Convert IOx Parquet files into InfluxDB Line Protocol format
```

Reply:
good call -- will change to `Parquet` and `line protocol`